Remotion/Web/Development.WebTesting/BrowserSession/BrowserSessionBase.cs

```diff
@@ -35,7 +35,7 @@ public abstract class BrowserSessionBase<T> : IBrowserSession
   {
     private const int c_driverShutdownWaitTime = 250;
     private const int c_browserShutdownWaitTime = 500;
-    private const int c_browserSubProcessShutdownWaitTime = 100;
+    private const int c_browserSubProcessShutdownWaitTime = 250;

```

Comment:
Should consider changing the times to TimeSpan, otherwise we don't know what the value means. Appending the unit is also possible, but not preferred. -> separate task

Reply:
Added https://github.com/re-motion/Trunk-SVN2Git/pull/49/commits/26b847a98c6f4b15888032133938af87427fe440.